stack/pico_stack.c

```diff
@@ -857,6 +856,20 @@ uint32_t pico_timer_add_hashed(pico_time expire, void (*timer)(pico_time, void *
 
 int MOCKABLE pico_stack_init(void)
 {
+
+#ifdef CHECK_MEM
+	//New log for malloc information 
+	fopen("mem_test.log", "w");
+
+	//Set rand seed
+	char *buf = getenv("MEM_TEST_SEED");
```

Comment:
Are you also putting this seed into some kind of log, so that we can try to recreate afterwards?